lib/crypto.ts

```diff
@@ -65,7 +65,12 @@ export function hexToBase64(hex: string): string {
 }
 
 export function toCSV(guest: Guest): string {
-  const values = [guest.name, guest.phone, guest.address, guest.resident]
+  const values = [
+    guest.name,
+    guest.phone,
+    `${guest.address}, ${guest.postalCode} ${guest.city}`,
```

Comment:
Yeah, I checked this because I had the same thought. But if it would break, it would already be broken if someone entered a `,` in the current adress field. Luckily for us, the `toCSV` method calls `JSON.stringify` on each field before joining them with `,`. So the result is nicely escaped, as you can also see in the test file: 

```js
  expect(decrypted).toBe('"Donnie","+11880","Täststr. 3, Köln",')
```